x-pack/plugins/security_solution/public/endpoint_alerts/store/middleware.ts

```diff
@@ -13,14 +13,34 @@ import {
 import { AlertConstants } from '../../../common/endpoint_alerts/alert_constants';
 import { ImmutableMiddlewareFactory } from '../../common/store';
 import { cloneHttpFetchQuery } from '../../common/utils/clone_http_fetch_query';
+
```

Comment:
I think we'll be removing this file eventually right? Not sure if that matters for the changes you've made.

Reply:
I think so, but it breaks the Resolver (by way of constantly sending it new alerts when the URL hash changes, which is "all the time" now) if these changes don't go with it.